threads/flask_api.py

```diff
@@ -63,19 +63,25 @@ def create_app(self):
         # TODO: Only accept ticks/pings from peers?
         @app.route('/forward/tick', methods=['POST'])
         def forward_tick():
-            if self.networker.stage == "select":
-                return "not accepting further ticks", 400
+            # if self.networker.stage == "select":
+            #     return "not accepting further ticks", 400
```

Comment:
reminder: to put a sanity_check(tick) to make sure rogues can't trigger permanent resyncs

Reply:
Added TODO